### PR TITLE
Add remove-members helper script

### PR DESCRIPTION
**Pull request by mrbobbytables**
This was originally opened in PR #2048  where it was desired that this script be replaced with a go version. Issue #2052 was created with that purpose, but hasn't been moved on yet. It's probably better that this get merged for now instead of living only on my local system. 😬 

It can be removed once the tool is written.

/cc @nikhita

### CODE REVIEW

**Review by nikhita — APPROVED** [review]
/lgtm

/hold
in case @cblecker has comments

/assign @cblecker

File: hack/remove-members.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly REPO_ROOT="$(git rev-parse --show-toplevel)"
+readonly CONFIG_PATH="$REPO_ROOT/config"
+readonly DRYRUN="${DRYRUN:-true}"
+
+members=()
+mapfile -t members < "$1"
```
Comment (spiffxp) [inline hack/remove-members.sh:25]: Can we get a brief usage if $1 is missing, and comment on expected contents/format of this file?

**Review by spiffxp — CHANGES_REQUESTED** [review]
Totes happy to merge and iterate away from bash, but would like some usage instructions

File: hack/remove-members.sh (hunk 1 above)
Comment (mrbobbytables) [reply hack/remove-members.sh:46]: Done, also added a check for its existence 👍

Comment (mrbobbytables) [reply hack/remove-members.sh:46]: oh derp, forgot 1 thing...1 sec

Comment (mrbobbytables) [reply hack/remove-members.sh:53]: okay, done for real :P

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
/hold cancel
merge and iterate

I would rather not have @mrbobbytables feel like they have to be the one to run this not-checked-in script next time we want to remove members

**Comment by blaisep** [comment]
Over in https://github.com/kubernetes/org/issues/2052 there's a request to implement this in a proper programming language and I wanted to offer to adapt a github action that some of us built over in:
https://github.com/cucumber/action-retire-inactive-contributors/actions/runs/4361276398/jobs/7625018582

The description of the features is in:
https://github.com/cucumber/action-retire-inactive-contributors/tree/main/features

and a general diagram is at:
https://github.com/cucumber/action-retire-inactive-contributors#now-as-a-github-action

**Comment by ryjones** [comment]
@blaisep - over at [Hyperledger](https://github.com/hyperledger), we're casting about for tooling like this. Shoot me an email?

**Pull request merged**